object.c

```diff
@@ -1664,7 +1664,7 @@ rb_true(VALUE obj)
  */
 
 
-static VALUE
+VALUE
```

Comment:
```suggestion
MJIT_FUNC_EXPORTED VALUE
```
To fix test failures, this function needs to be exported.